ui/src/app/shared/formly/formly-skeleton-wrapper.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, Input } from '@angular/core';
+import { FormGroup } from '@angular/forms';
+import { FormlyFieldConfig } from '@ngx-formly/core';
+
+
+/// to use the skeleton, simply pass the form data to this component selectoro with a boolean indicator to show information.
```

Comment:
Better would be: 

This wrapper is used to display a loading animation for a line until the async call is finished, the @Input show is true, respectively.

@Input show: is used to determine when the async call has finished
@Input fields the formlyfields
@Input form the formGroup
@Input model the model